homeassistant/components/permobil/sensor.py

```diff
@@ -209,6 +225,15 @@ def __init__(
             f"{coordinator.p_api.email}_{self.entity_description.key}"
         )
 
+    @property
+    def native_unit_of_measurement(self) -> str | None:
```

Comment:
Can we set `_attr_native_unit_of_measurement` in the entity init method instead or may the unit change during the entity lifetime?

Reply:
Oh that's a good point, haven't thought of this idea